tests/RobotFramework/tests/cumulocity/smartrest_one/smartrest_one.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Resource            ../../../../resources/common.resource
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Test Teardown       Custom Teardown
+
+Test Tags           theme:c8y    theme:operation
+
+
+*** Variables ***
+${SMART_REST_ONE_TEMPLATES}=
+...                             SEPARATOR=\n
+...                             10,339,GET,/identity/externalIds/c8y_Serial/%%,,application/vnd.com.nsn.cumulocity.externalId+json,%%,STRING,
+...                             10,311,GET,/alarm/alarms?source\=%%&status\=%%&pageSize\=100,,,%%,UNSIGNED STRING,
+...                             11,800,$.managedObject,,$.id
+...                             11,808,$.alarms,,$.id,$.type
+
+
+*** Test Cases ***
+Supports SmartREST 1.0 Templates
+    [Template]    Register and Use SmartREST 1.0. Templates
+    use_builtin_bridge=true
+    use_builtin_bridge=false
+
+
+*** Keywords ***
+Register and Use SmartREST 1.0. Templates
```

Comment:
The registration part can be extracted as a keyword into our `robotframework-c8y` in a separate PR along with the`Delete SmartREST 1.0 Template` that was also added separately. The registration keyword can also auto cleanup of the registered  templates as part of test teardown.